tests/integration/index.test.ts

```diff
@@ -0,0 +1,74 @@
+import { ANT, AOProcess, ArweaveSigner, createAoSigner } from '@ar.io/sdk';
+import { strict as assert } from 'node:assert';
+import * as fs from 'node:fs';
+import { describe, it } from 'node:test';
+
+import {
+  AO_LOADER_HANDLER_ENV,
+  AO_LOADER_OPTIONS,
+  LocalAO,
+  TEST_AOS_ANT_WASM,
+} from './utils.js';
+
+const testWalletJSON = fs.readFileSync('./test-wallet.json', {
+  encoding: 'utf-8',
+});
+
+const testWallet = JSON.parse(testWalletJSON);
+const signers = [
+  new ArweaveSigner(testWallet),
+  createAoSigner(new ArweaveSigner(testWallet)),
+] as const;
+
+describe('integration esm tests', async () => {
+  async function createLocalANT() {
+    return new AOProcess({
+      processId: 'ant-'.padEnd(43, '1'),
+      ao: (await LocalAO.init({
+        wasmModule: TEST_AOS_ANT_WASM,
+        aoLoaderOptions: AO_LOADER_OPTIONS,
+        handlerEnv: AO_LOADER_HANDLER_ENV,
+      })) as any,
+    });
+  }
```

Comment:
nit: move this into utils and import in the tests

Reply:
https://github.com/ar-io/ar-io-sdk/pull/387/commits/3a583d7de2f539c0e93d8a13a1112b1d28d8086e